flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/KubernetesOperatorConfigOptions.java

```diff
@@ -458,4 +458,12 @@ public static String operatorConfigKey(String key) {
                     .defaultValue(Duration.ofDays(1))
                     .withDescription(
                             "Time after which jobmanager pods of terminal application deployments are shut down.");
+
+    @Documentation.Section(SECTION_ADVANCED)
+    public static final ConfigOption<Duration> CANARY_RESOURCE_TIMEOUT =
+            operatorConfig("health.canary.resource.timeout")
```

Comment:
This does not match the config described in the documentation: `health.canary.deployment.timeout`

Reply:
thanks, I need to regenerate the docs :)